lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/TabView.js

```diff
@@ -23,6 +23,10 @@ qx.Class.define("callbackery.ui.TabView", {
             var page = tabMap[cfg.name]
                 = new callbackery.ui.Page(cfg);
             this.add(page);
+            console.log('Registering page', cfg.name);
```

Comment:
no console output in production apps .... you could use `this.debug` if you want. Although this is probably a leftover.